diff --git a/vllm/__init__.py b/vllm/__init__.py
index 968d1a1..aa1d865 100644
--- a/vllm/__init__.py
+++ b/vllm/__init__.py
@@ -1,3 +1,4 @@
+import amdsmi  # gfx1151 workaround (see docs/troubleshooting.md#amdsmi)
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: Copyright contributors to the vLLM project
 """vLLM: a high-throughput and memory-efficient inference engine for LLMs"""
